scripts/verify-replace.sh

```diff
@@ -16,27 +19,45 @@ do
     echo                                                                     
     echo =========================================================================================                                            
     repo_path=${BASE_REPO_PATH}/$(basename ${repo})
+    ERRFILE=$(mktemp ${TMP_DIR}$(basename ${repo}).XXX)
```

Comment:
IMHO, the file extension could be something more descriptive, like `.errors`.

Reply:
It would be okay if the exact same named files are created in CI because these are tmp files which are automatically deleted once the workspace is deleted after each ci run. 
But if this same script someone runs locally, its created in /tmp folder, and it gets deleted once the local env restarts. if someone does 2-3 runs locally without restarting the local, it will be a problem
I did think about deletion of files, but didn't want to deal with the risk of deleting something accidentally.
This was the reason to have the Random word generated as the file extension so that the file names are always unique.
But i do agree to have a self explanatory file  name and hence added -error in the file name rather than extension
eg.`(toolchain-common-error.b3m)`